app/src/main/java/io/intrepid/contest/screens/sendinvitations/selectcontacts/SelectContactsActivity.java

```diff
@@ -97,6 +100,26 @@ protected int getLayoutResourceId() {
     }
 
     @Override
+    protected void onViewCreated(Bundle savedInstanceState) {
+        super.onViewCreated(savedInstanceState);
+
+        setActionBarTitle(ACTION_BAR_TITLE);
+        setActionBarDisplayHomeAsUpEnabled(true);
+    }
+
+    @Override
+    public boolean onCreateOptionsMenu(Menu menu) {
+        MenuItem searchItem = menu.add(R.string.common_search);
+        searchItem.setIcon(R.drawable.search_icon);
+        searchItem.setShowAsAction(MenuItem.SHOW_AS_ACTION_IF_ROOM);
+
+        SearchView searchView = new SearchView(this);
+        searchView.setOnQueryTextListener(this);
+        searchItem.setActionView(searchView);
```

Comment:
does the searchView need an id? i would imagine when the user searches, you make a call to the presenter